Capstone/Models/CollegeNews.m

```diff
@@ -19,8 +19,10 @@ - (instancetype)initWithDictionaryNews:(NSDictionary *)dictionary {
 + (NSMutableArray *)collegesWithArrayNews:(NSArray *)dictionaries {
     NSMutableArray *collegeNews = [[NSMutableArray alloc] init];
     for (NSDictionary *dictionary in dictionaries) {
-        CollegeNews *college = [[CollegeNews alloc] initWithDictionaryNews:dictionary];
-        [collegeNews addObject:college];
+        if (dictionary[@"title"] != [NSNull null] && dictionary[@"description"] != [NSNull null] && dictionary[@"urlToImage"] != [NSNull null]) {
```

Comment:
think you should be able to just do
```
dictionary[@"title"] != null
```
right?

Reply:
{          
   "title":"Some news",
   "description":null,
   "urlToImage":"www.sth.com"
}

That was what I also originally assumed. However, in NSDictionary, you can only represent "null" values with NSNull.
https://developer.apple.com/documentation/foundation/nsdictionary?language=objc